### DISCUSSION TITLE
Does MinIO support automatic healing for files deleted on one server in a replicated setup?

### DISCUSSION THREAD

**Discussion by faryar76** [minio/minio#17143] [Q&A]
Hello MinIO community,

I have set up two MinIO servers in a single node single drive mode, and they are connected to each other through site replication. Both servers are located in different locations. I am wondering if MinIO supports automatic healing for files deleted on one server in this replicated setup.

For example, if I delete a file on server 1, will it automatically heal from server 2, which is located in another location? Or do I need to manually replicate the file to ensure its availability on both servers?
(i mentioned before only 1 disk to each side )
I would greatly appreciate any insights or guidance on this matter.

Thank you in advance.

**Comment by harshavardhana** [comment]
You would have to manually resync
